vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/CredentialAuthorizationServiceStrategy.kt

```diff
@@ -34,7 +41,7 @@ class CredentialAuthorizationServiceStrategy(
         }
         .joinToString(" ")
 
-    override fun filterAuthorizationDetails(authorizationDetails: Set<AuthorizationDetails>) =
+    override fun filterAuthorizationDetails(authorizationDetails: Collection<AuthorizationDetails>) =
```

Comment:
This function here does not check for duplicates

Reply:
But it returns a set!?

Reply:
🤦